azure-pipelines.yml

```diff
@@ -0,0 +1,69 @@
+trigger:
+  branches:
+    include:
+      - cccs-*
+      - feature/CLDN-*
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  tag: '$(Build.BuildId)'
+
+stages:
+- stage: Build_and_Publish_Npm_Artifacts
+  displayName: Build, package and publish npm artifacts.
+  jobs:
+  - job: BuildNpmArtifacts
+    displayName: Build, package and publish npm artifacts.
+    steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '14.x'
+    - script: |
+        yarn install --frozen-lockfile
+      displayName: 'Install dependencies'
+    - script: |
+        yarn build
+      displayName: 'Build packages'
+    - script: |
+        yarn test
+      displayName: 'Run unit tests'
+  - job: PublishArtifactsToFeed
+    dependsOn: BuildNpmArtifacts
+    steps:
+    - template : publish-template.yml
+      parameters:
+        workingDirs: [
```

Comment:
This JSON-style list is kind of weird in a YAML file. Can it be converted to YAML?